packages/elements-core/src/components/TryIt/Body/request-body-utils.ts

```diff
@@ -10,6 +10,7 @@ import {
 } from '../Parameters/parameter-utils';
 
 export type BodyParameterValues = Record<string, string | File>;
+export type ParamaterOptional = Record<string, boolean>;
```

Comment:
There's a typo here. Let's rename it.

```suggestion
export type ParameterOptional = Record<string, boolean>;
```